### ISSUE TITLE
无法打开PDF文档

### ISSUE THREAD

**Issue by EvanMeek** [manateelazycat/emacs-application-framework#383]
**Describe the bug**
无法打开pdf文件

**To Reproduce**

**Expected behavior**
使用M-x eaf-open 打开PDF文档报错

**Versions (please complete the following info):**
 - Distro and DE/WM: Archlinux 5.8.1 XFCE4
 - Versions of Dependencies:
`pip list:`
`➜  ~ pip list
Package                Version
---------------------- -----------
ag                     1.0
altgraph               0.17
appdirs                1.4.4
arandr                 0.1.10
asn1crypto             1.4.0
astroid                2.3.3
attrs                  19.3.0
autopep8               1.5
beautifulsoup4         4.9.1
black                  19.10b0
btrfsutil              1.2.0
CacheControl           0.12.6
cairocffi              1.1.0
cffi                   1.14.2
chardet                3.0.4
chrome-gnome-shell     0.0.0
cli-helpers            2.1.0
click                  7.1.2
colorama               0.4.3
configobj              5.0.6
contextlib2            0.6.0.post1
cryptography           3.0
dbus-python            1.2.16
decorator              4.4.2
direnv                 2019.4.13
distlib                0.3.1
distro                 1.5.0
dnspython              1.16.0
docopt                 0.6.2
entrypoints            0.3
env-file               2019.4.13
feedparser             5.2.1
filelock               3.0.12
flake8                 3.7.9
flake8-blind-except    0.1.1
Flask                  1.1.2
future                 0.18.2
get                    2019.4.13
greenlet               0.4.16
grip                   4.5.2
html5lib               1.1
httplib2               0.18.1
idna                   2.10
importlib-metadata     1.6.1
isort                  4.3.21
itsdangerous           1.1.0
jedi                   0.15.2
jieba                  0.42.1
Jinja2                 2.11.2
json-rpc               1.13.0
lanzou-api             2.5.0
lazy-object-proxy      1.4.3
lensfun                0.3.95
lit                    0.10.1.dev0
lxml                   4.5.2
Markdown               3.2.2
MarkupSafe             1.1.1
mccabe                 0.6.1
Menu                   3.2.2
more-itertools         8.4.0
msgpack                1.0.0
musicbox               0.1.2.4
mycli                  1.22.1
mysql-connector-python 8.0.19
nose                   1.3.7
numpy                  1.18.4
OpenCC                 1.1.1
ordered-set            4.0.2
packaging              20.4
PacVis                 0.2.7
parso                  0.6.2
path-and-address       2.0.1
pathspec               0.7.0
pep517                 0.8.2
Pillow                 7.2.0
pip                    20.1.1
pipsi                  0.9
pluggy                 0.13.1
ply                    3.11
post                   2019.4.13
progress               1.5
promise                2.3
prompt-toolkit         3.0.6
psutil                 5.7.2
public                 2019.4.13
pwquality              1.4.2
py                     1.8.1
py3status              3.28
pyalpm                 0.9.1
pyalsa                 1.1.6
pycairo                1.19.1
pycodestyle            2.5.0
pycparser              2.20
pycurl                 7.43.0.5
pydflatex              0.0.0
pydocstyle             5.0.2
pyflakes               2.1.1
pygame                 1.9.6
Pygments               2.6.1
PyGObject              3.36.1
pyinotify              0.9.6
PyInstaller            3.6
pylint                 2.4.4
PyMuPDF                1.17.5
PyMySQL                0.10.0
pynvim                 0.4.0
pyparsing              2.4.7
PyQt5                  5.15.0
PyQt5-sip              12.8.0
PyQtWebEngine          5.15.0
pyte                   0.8.0
pytest                 5.4.1
python-dotenv          0.13.0
python-jsonrpc-server  0.3.4
python-language-server 0.31.9
python-xlib            0.26
pytoml                 0.1.21
pyzmq                  19.0.1
qrcode                 6.1
qtile                  0.16.1
query-string           2019.4.13
ranger-fm              1.9.3
regex                  2020.2.20
request                2019.4.13
requests               2.24.0
requests-toolbelt      0.9.1
resolvelib             0.4.0
retrying               1.3.3
rope                   0.16.0
screenkey              1.2
service-factory        0.1.6
setuptools             49.6.0
sip                    4.19.23
six                    1.15.0
snowballstemmer        2.0.0
soupsieve              2.0.1
sqlparse               0.3.1
tabulate               0.8.7
team                   1.0
termcolor              1.1.0
terminaltables         3.1.0
terminator             1.92
thefuck                3.30
tldr                   1.0.0
toml                   0.10.1
tornado                6.0.4
typed-ast              1.4.1
ujson                  1.35
urllib3                1.25.10
values                 2019.4.13
variety                0.8.3
virtualenv             20.0.23
wakatime               13.0.3
wcwidth                0.2.5
webencodings           0.5.1
Werkzeug               1.0.1
wheel                  0.34.2
wrapt                  1.11.2
xcffib                 0.10.1
yapf                   0.29.0
you-get                0.4.1432
youtube-dl             2020.2.16
zipp                   3.1.0`
 - M-x emacs-version: GNU Emacs 27.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.22) of 2020-08-14

**Screenshots**

![emacs-eaf-pdf-error gif](https://user-images.githubusercontent.com/27056033/90608442-3d6e3c00-e235-11ea-9fc7-f424d2514a19.png)

**Additional context**
看报错应该是fitz库有问题，但是不知道如何解决

**Comment by MatthewZMD** [comment]
可以试试重新下载

**Comment by manateelazycat** [comment]
按照命令

```
pip install fitz frontend  
pip install PyMuPDF==1.17.4
```

重装会解决问题。

**Comment by manateelazycat** [comment]
按照上游作者的提示： https://github.com/pymupdf/PyMuPDF/issues/523

更好的方式是，先卸载系统的pymupdf和fitz

```
sudo pacman -R python-pymupdf
pip uninstall fitz
```

然后在通过 pip 安装最新的 pymupdf

```
pip install pymupdf
```

就可以解决这个问题。

**Comment by improve100** [comment]
ubuntu1604  可以试试这个sudo pip3 install pymupdf==1.18.0
